otter/log/formatters.py

```diff
@@ -237,6 +241,19 @@ def audit_log_formatter(eventDict, timestamp, hostname):
     return audit_log_params
 
 
+@singledispatch
+def serialize_to_jsonable(obj):
+    """
+    Serialize any object to a JSONable form
+    """
+    return None
+
+
+# serialize base types
+for t in (list, tuple, basestring, int, float, long):
+    serialize_to_jsonable.register(t, identity)
```

Comment:
I guess you missed some of my later comments :) We can't just use `identity` for container types, since they may contain non-jsonable objects, so we actually need to recurse on their contents.

I think it's fine if you want to skip this entirely for this PR, since we're only using this function for exceptions anyway.